python/pyspark/sql/functions/builtin.py

```diff
@@ -233,6 +233,7 @@ def lit(col: Any) -> Column:
 def col(col: str) -> Column:
     """
     Returns a :class:`~pyspark.sql.Column` based on the given column name.
+    An alias of :func:`column`.
```

Comment:
If you'd like to do this, you should better create the dedicated docstring for `column`, and say `column` is an alias.